bin/Cargo.toml

```diff
@@ -71,7 +71,7 @@ name = "named"
 path = "src/named.rs"
 
 [dependencies]
-clap = "2.33"
+clap = {version = "3.0", features = ["cargo"] }
```

Comment:
Spaces inside braces for consistency?

Reply:
Probably just a mistake. I’ll update. I’m currently fighting clap on a minimum rust version.

might need to update to 1.52 if we want this, but maybe even more recent. I’ll know in a minute.

Reply:
clap is 1.54, but I filed an issue upstream, should be easy to get it down to 1.52.